addons/web/static/tests/views/fields/one2many_field.test.js

```diff
@@ -13226,3 +13248,49 @@ test("expand record in dialog", async () => {
     await contains(".o_dialog .modal-header .o_expand_button").click();
     expect.verifySteps([[4, "turtle", "ir.actions.act_window", [[false, "form"]]]]);
 });
+
+test("one2many kanban: add button kanban's card only with no control", async () => {
+    Partner._fields.control = fields.One2many({
+        string: "one2many field",
+        relation: "partner",
+        relation_field: "trululu",
+    });
+    Partner._records[0].p = [4];
+    Partner._records[0].control = [4];
+
+    await mountView({
+        type: "form",
+        resModel: "partner",
+        arch: `
+            <form>
+                <field name="p">
+                    <kanban>
+                        <templates>
+                            <t t-name="card">
+                                <field name="name"/>
+                            </t>
+                        </templates>
+                    </kanban>
+                </field>
+                <field name="control">
+                    <kanban>
+                        <control>
+                            <create string="Add Custom" />
+                        </control>
+                        <templates>
+                            <t t-name="card">
+                                <field name="name"/>
+                            </t>
+                        </templates>
+                    </kanban>
+                </field>
+            </form>`,
+        resId: 1,
+    });
+
+    expect("[name='p'] .o_x2m_control_panel .o-kanban-button-new").toHaveCount(0);
+    expect("[name='p'] .o_kanban_renderer .o-kanban-button-new").toHaveCount(1);
+
+    expect("[name='control'] .o_x2m_control_panel .o-kanban-button-new").toHaveCount(0);
+    expect("[name='control'] .o_kanban_renderer .o-kanban-button-new").toHaveCount(0);
```

Comment:
I know (or I guess) it's already tested in another test, but for the sake of completeness of this test, I would add a custom classname on the control, and assert it is there